index.md

```diff
@@ -1,11 +1,14 @@
 
 ## Product Management at 18F
 
-As a product manager at 18F, you may be put in a client-facing role to help a government partner build a mission critical product.  You may also be working on one of TTS’s flagship products, like <a href="https://eregs.github.io/" target="_blank">eRegs</a>, <a href="https://federalist.18f.gov" target="new">Federalist</a>, <a href="https://cloud.gov" target="new">cloud.gov</a> or <a href="https://login.gov/" target="_blank">login.gov</a>. In all cases, you will create or steward the product vision and strategy, and drive a cross-functional team to deliver the right product to the right audience. We will talk more about this in the [Defining the future state]({{ "/future/" | relative_url }}) section.
+As a product manager at 18F, you may be put in a client-facing role to help a government partner build a mission critical product.  You may also be working on one of TTS’s flagship products, like <a href="https://eregs.github.io/" target="_blank">eRegs</a>, <a href="https://federalist.18f.gov" target="new">Federalist</a>, <a href="https://cloud.gov" target="new">cloud.gov</a> or <a href="https://login.gov/" target="_blank">login.gov</a>. In all cases, you will create or steward the product vision and strategy, and drive a cross-functional team to deliver the right product to the right audience. We will talk more about this in the [Defining the future state]({{site.baseurl}}/define/) section.
 
-**You will also focus on sustainability.** Anything we build should empower our partner agencies to better meet their mission, be relatively easy for them to adopt and support, and be easy for end users to use. If we are building something with a partner, the product needs to be constructed and hosted in a way that the partner can manage, and the partner should be empowered to take over after we leave. We'll talk more about this in the [Enable partners for long-term success]({{ "/partners/enable/" | relative_url }}) section.
\ No newline at end of file
+**You will also focus on sustainability.** Anything we build should empower our partner agencies to better meet their mission, be relatively easy for them to adopt and support, and be easy for end users to use. If we are building something with a partner, the product needs to be constructed and hosted in a way that the partner can manage, and the partner should be empowered to take over after we leave. We'll talk more about this in the [Enable partners for long-term success]({{site.baseurl}}/partners/) section.
```

Comment:
I think the `relative_url` filter [is preferable now](https://ben.balter.com/jekyll-style-guide/config/#url) to calling `site.baseurl`, but if it's working like this, it probably doesn't matter that much.